### PR TITLE
Update language bindings

### PR DESCRIPTION
**Pull request by Isopod**

### CODE REVIEW

File: bindings/rust/lib.rs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+//!
+//! [Language]: https://docs.rs/tree-sitter/*/tree_sitter/struct.Language.html
+//! [language func]: fn.language.html
+//! [Parser]: https://docs.rs/tree-sitter/*/tree_sitter/struct.Parser.html
+//! [tree-sitter]: https://tree-sitter.github.io/
+
+use tree_sitter::Language;
+
+extern "C" {
+    fn tree_sitter_quick_check() -> Language;
```
Comment (ObserverOfTime) [inline bindings/rust/lib.rs:21]: `s/quick_check/pascal/g` :thinking:

**Review by ObserverOfTime — COMMENTED** [review]
You should check the generated bindings and fix things that are not handled automatically like versions and links. You might also want to add some [workflows].

[workflows]: https://github.com/tree-sitter/workflows

File: bindings/rust/lib.rs (hunk 1 above)
Comment (Isopod) [reply bindings/rust/lib.rs:21]: Thanks for stopping by, much appreciated!

I did notice that the version numbers were a bit weird, but I cross-checked with some official Tree-Sitter grammars and they [looked the same](https://github.com/tree-sitter/tree-sitter-python/blob/b466ffb4d7ce0a833754429ad8d3a02e950ed74c/Makefile#L1), so I thought it was supposed to be this way. No idea how the "quick_check" got in there.

I hope I don't have to manually update a gazillion files every time I bump the version number. Guess I'll have to familiarize myself with GitHub workflows.

Comment (ObserverOfTime) [reply bindings/rust/lib.rs:21]: > I hope I don't have to manually update a gazillion files every time I bump the version number.

Unfortunately you do, until the CLI gets a command for it. But it doesn't matter if you don't publish packages.

**Pull request merged**